<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Oscilloscope</class>
 <widget class="QWidget" name="Oscilloscope">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="plotWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QListWidget" name="plotsListWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="addPlotBtn">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delPlotBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="renPlotBtn">
         <property name="text">
          <string>重命名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clrPlotBtn">
         <property name="text">
          <string>清空图表</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
        <property name="horizontalSpacing">
         <number>9</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>单次数据数目</string>
          </property>
          <property name="buddy">
           <cstring>singleTimeItemNumBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="singleTimeItemNumBox">
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
          <property name="value">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="manualAddDataBtn">
          <property name="text">
           <string>添加数据</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="manualAddDataBox">
          <property name="maximum">
           <number>4095</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>plotsListWidget</tabstop>
  <tabstop>addPlotBtn</tabstop>
  <tabstop>delPlotBtn</tabstop>
  <tabstop>renPlotBtn</tabstop>
  <tabstop>singleTimeItemNumBox</tabstop>
  <tabstop>manualAddDataBox</tabstop>
  <tabstop>manualAddDataBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
